file_operations/src/main/java/com/amaze/filemanager/fileoperations/filesystem/OpenMode.java

```diff
@@ -30,6 +30,8 @@ public enum OpenMode {
   FILE,
   SMB,
   SFTP,
+  FTP,
+  NFS,
```

Comment:
Probably a description for these two should be added, explaining what it is, and what the difference to SFTP is, for clarity.